company_website/views.py

```diff
@@ -1,10 +1,14 @@
+import os
 from typing import Any
 
 from django.conf import settings
 from django.contrib import messages
+from django.http import HttpRequest
+from django.http import HttpResponse
 from django.urls import reverse_lazy
 from django.views.generic import FormView
 from django.views.generic import ListView
+from rest_framework.decorators import api_view
```

Comment:
No, no, no :D We shouldn't use it here. We should uninstall this package from our project at all. Please use different way to accomplished this check but don't use rest_framework :). Please also delete it from project :)